custom/icds_reports/sqldata/exports/growth.py

```diff
@@ -0,0 +1,179 @@
+from corehq.apps.locations.models import SQLLocation
```

Comment:
File name could be more specific. may be something like `growth_tracker_report.py`

Reply:
updated!